features/new-look/data.json

```diff
@@ -0,0 +1,16 @@
+{
+    "title": "New Look",
+    "description": "Changes the look of the Scratch Site to a more modern, fresh look.",
+    "credits": [
+      { "username": "LifeCoderBoy", "url": "https://scratch.mit.edu/users/LifeCoderBoy/" },
+      { "username": "_nix", "url": "https://scratch.mit.edu/users/_nix/" }
+    ],
+    "styles": [{ "file": "scratch-www.css", "runOn": "/" }, { "file": "scratch-editor.css", "runOn": "/projects/*" }, { "file": "scratch-profile.css", "runOn": "/users/*" }],
+    "scripts": [{"file": "scratch-www.js", "runOn": "/"}, { "file": "scratch-profile.js", "runOn": "/users/*" }],
+    "tags": ["Theme", "New"],
+    "type": ["Website", "Editor"],
+    "options": [{ "id": "theme", "name": "Mode", "type": 0 }],
```

Comment:
You have an option yet it is never referenced anywhere.

Reply:
See scratch-www.js line 4:
ScratchTools.Storage.theme
Isn't that how you reference an option.

Reply:
@rgantzos Also, instead of integrating dark mode into my "tool" why can't me (or someone else) just make another dark mode tool that is compatible with this?